deployments/helm/configmapfiles/discovery-engine/conf.yaml

```diff
@@ -24,6 +24,21 @@ application:
     fromsource-filter:
       - "knoxAutoPolicy"
 
+  admission-controller:
+    generic-policy-list:
+      - "restrict-deprecated-registry"
+      - "prevent-cr8escape"
+      - "check-kernel-version"
+      - "restrict-ingress-defaultbackend"
+      - "restrict-nginx-ingress-annotations"
+      - "restrict-ingress-paths"
+      - "prevent-naked-pods"
+      - "restrict-wildcard-verbs"
+      - "restrict-wildcard-resources"
+      - "require-requests-limits"
+      - "require-pod-probes"
+      - "drop-cap-net-raw"
+

```

Comment:
Configurable list, only below mentioned generic policies would be generated.

Reply:
These are generic policies applicable to the cluster (`ClusterPolicy`). Configuring them for workloads would make them workload associated whereas here we are targeting the whole cluster.

For policies on `pods`, they would be modified when you apply them to match on higher level resources. https://kyverno.io/docs/writing-policies/autogen/
```yaml
  - match:
      any:
      - resources:
          kinds:
          - Pod
```